docs(README): add deployment section (#385)
hossein-zare/react-native-dropdown-picker: README.md

**Donate Bitcoin**
+ **Payment Address:** `bc1q7sgzww3l0fk3dyy9mzcjxdw9rla3yvhuyf0qku`  
![QR Code](https://user-images.githubusercontent.com/56504893/116758583-a0031300-aa25-11eb-9624-0009346d2290.png)

# Merge and Release Process

## Branches in use

### Development

PRs should be made against and merged into the [`dev-5.x`](https://github.com/hossein-zare/react-native-dropdown-picker) branch, which is set as the `default` branch on github.

### Release

Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.

> To make a new release, follow these steps:

Author: ```suggestion
## Release Process

To make a new release, follow these steps:
```

> * Verify the `dev-5.x` branch has all the changes desired in a release and works well
> * Make and merge a final PR into `dev-5.x` that increments the version number in `package.json`
> * Make and merge a PR from the `dev-5.x` branch to the `5.x` branch
> * Using the GitHub web UI, draft a new release using tag name `v5.x.x` (replace the `x` values as appropriate of course), against target `5.x`, with release name `v5.x.x` (again, with appropriate numbers in place of `x` of course)

Author: ```suggestion
* Verify the development branch has all the changes desired in a release and works well
* Make and merge a final PR into development branch that increments the version number in `package.json`
* Make and merge a PR from the development branch to the release branch
* Using the GitHub web UI, draft a new release using tag name `vx.x.x` (replace the `x` values as appropriate of course), with the release branch as the target, with release name `vx.x.x` (again, with appropriate numbers in place of `x` of course)
```

**Donate Bitcoin**
+ **Payment Address:** `bc1q7sgzww3l0fk3dyy9mzcjxdw9rla3yvhuyf0qku`  
![QR Code](https://user-images.githubusercontent.com/56504893/116758583-a0031300-aa25-11eb-9624-0009346d2290.png)

# Merge and Release Process

## Branches in use

### Development

PRs should be made against and merged into the [`dev-5.x`](https://github.com/hossein-zare/react-native-dropdown-picker) branch, which is set as the `default` branch on github.

### Release

> Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.

Reviewer: There's a typo "teh" :)

Author: :facepalm: and that's why review never ever hurts :-). Great spot

> Releases are currently made from teh [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.

Author: ```suggestion
Releases are currently made from the [`5.x`](https://github.com/hossein-zare/react-native-dropdown-picker/tree/5.x) branch.
```
